app/uk/gov/hmrc/helptosavefrontend/models/userinfo/NSIPayload.scala

```diff
@@ -22,22 +22,26 @@ import java.time.format.DateTimeFormatter
 import cats.instances.string._
 import cats.syntax.eq._
 import play.api.libs.json._
-import uk.gov.hmrc.helptosavefrontend.models.userinfo.NSIUserInfo.ContactDetails
+import uk.gov.hmrc.helptosavefrontend.forms.BankDetails
+import uk.gov.hmrc.helptosavefrontend.models.userinfo.NSIPayload.ContactDetails
 
 import scala.util.{Failure, Success, Try}
 
-case class NSIUserInfo(forename:            String,
-                       surname:             String,
-                       dateOfBirth:         LocalDate,
-                       nino:                String,
-                       contactDetails:      ContactDetails,
-                       registrationChannel: String         = "online")
-
-object NSIUserInfo {
-
-  implicit class NSIUserInfoOps(val nsiUserInfo: NSIUserInfo) {
-    def updateEmail(newEmail: String): NSIUserInfo =
-      nsiUserInfo.copy(contactDetails = nsiUserInfo.contactDetails.copy(email = newEmail))
+case class NSIPayload(forename:            String,
```

Comment:
Can we stick we NSIUserInfo as the name? As payload doesn't really say what it is